src/formatMarkup.js

```diff
@@ -1,31 +1,70 @@
+// @ts-check
+/** @import { DefaultTreeAdapterMap } from "parse5" */
+
 import { parseFragment } from 'parse5';
 
 import { logger } from '@/helpers.js';
 
+const VOID_ELEMENTS = Object.freeze([
```

Comment:
I don't really care if these are outside of the `diffableFormatter` function or inside of it. I think it's fine either way. BUT, eventually I want to have a live demo of this formatter on the docs site (whenever that gets made), and it would be simpler if everything needed by this component was self-contained (expcet the parse5 import).

Example:

https://jsfiddle.net/tyL7b06n

Reply:
Changed my mind, started on the docs site and I'll be using a build system, so I'm going to just import the whole library in directly and use it on the page.

* https://TheJaredWilcurt.com/vue-snapshot-serializer